packages/web-components/components/accordion-group.ts

```diff
@@ -0,0 +1,8 @@
+import { defineCustomElement } from 'vue';
+import { PuikAccordionGroup } from '@prestashopcorp/puik-components';
+import type { CustomElementWithTagName } from '../types';
+
+const PuikAccordionGroupCe = defineCustomElement(PuikAccordionGroup) as unknown as CustomElementWithTagName;
+PuikAccordionGroupCe.tagName = 'puik-accordion-group-ce';
+
+export default PuikAccordionGroupCe;
```

Comment:
I'm wondering if finally we should create a directory by web component. For now we've only one TypeScript file by Web Component but later if there's modification to do like CSS or vanilla JS, we'll need a directory.

Reply:
normally all modifications and css import are done in the vue-component part (the goal being to have a single source in order to avoid different behavior between vue and web-components). There should be no other files

Reply:
Ok for now. We'll see later if we need a directory by component or not